src/components/general/NotificationsSideSheet/NotificaitonCardWrapper.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { NotificationCard, useNotificationCardActions } from '@equinor/fusion';
+import {
+    StandardNotificationCard,
+    useElevationClassName,
+    Button,
+    Spinner,
+    IconButton,
+    DeleteIcon,
+} from '@equinor/fusion-components';
+import classNames from 'classnames';
+import * as styles from './styles.less';
+
+type NotificationCardWrapperProps = {
+    notification: NotificationCard;
+};
+
+const NotificationCardWrapper: React.FC<NotificationCardWrapperProps> = ({ notification }) => {
+    const {
+        isMarkingNotification,
+        markNotificationsAsSeenAsync,
+        deleteNotificationCard,
+        isDeletingNotification,
+    } = useNotificationCardActions(notification);
+    
+    const isSeen = React.useMemo(() => notification.seenByUser, [notification]);
```

Comment:
Does this need to be memoized? A bool will not trigger rerender as long as it's the same value